### PR TITLE
revisions

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/recharge__charge_line_item_history.sql [hunk 1]
```diff
@@ -93,10 +93,10 @@ with charges as (
 ), joined as (
     select
         unioned.charge_id,
-        {# unioned.index, #} --
         row_number() over(partition by unioned.charge_id 
             order by unioned.line_item_type, unioned.index) 
-            as charge_row_num, --thinking of this to replace index.
+            as charge_row_num, -- CLARIFY IN DOCS
```
Comment (fivetran-joemarkiewicz) [inline models/recharge__charge_line_item_history.sql:98]: Reminder to remove after updating the docs.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-catfritz great job reworking the MRR report to take into consideration the subscriptions prepaid and rendered over time! I have just two very minor comments about documenting a field, but everything else looks great and validated on my end!

File: models/recharge__charge_line_item_history.sql (hunk 1 above)
Comment (fivetran-catfritz) [reply models/recharge__charge_line_item_history.sql]: done

File: models/recharge.yml [hunk 2]
```diff
@@ -75,19 +75,26 @@ models:
...
+            - charge_row_num
     columns:
       - name: charge_id
         description: "{{ doc('charge_id') }}"
         tests:
           - not_null
-      - name: index
-        description: A unique numeric row produced for every concurrent charge_id.
+      - name: charge_row_num
+        description: EXPLAIN
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply models/recharge.yml]: Updated.

**Comment by fivetran-catfritz** [comment]
@fivetran-joemarkiewicz I have updated the docs (the "Files changed" suppresses recharge.yml) and renamed some columns for clarity. It is ready for re-review!

File: dbt_project.yml [hunk 3]
```diff
@@ -6,7 +6,6 @@ require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 vars:
   recharge:
-    dbt_date:time_zone: "UTC"
```
Comment (fivetran-catfritz) [inline dbt_project.yml:9]: Note--this was leftover from some prior experiments but unnecessary now.

File: models/intermediate/int_recharge__customer_details.sql [hunk 4]
```diff
@@ -11,15 +11,12 @@ with customers as (
...
+        round(cast(sum(order_total_price) as {{ dbt.type_numeric() }}), 2) as total_amount_ordered,
+        round(cast(avg(order_total_price) as {{ dbt.type_numeric() }}), 2) as avg_order_amount,
+        round(cast(sum(order_item_quantity) as {{ dbt.type_numeric() }}), 2) as total_quantity_ordered,
         round(cast(avg(order_item_quantity) as {{ dbt.type_numeric() }}), 2) as avg_item_quantity_per_order,
         round(cast(sum(order_line_item_total) as {{ dbt.type_numeric() }}), 2) as total_order_line_item_total,
-        round(cast(avg(order_line_item_total) as {{ dbt.type_numeric() }}), 2) as avg_order_line_item_total,
-        round(cast(sum(total_tax) as {{ dbt.type_numeric() }}), 2) as total_amount_taxed,
-        round(cast(sum(total_discounts) as {{ dbt.type_numeric() }}), 2) as total_amount_discounted,
-        round(cast(sum(total_refunds) as {{ dbt.type_numeric() }}), 2) as total_refunds,
-        count(case when lower(billing.order_type) = 'checkout' then 1 else null end) as total_one_time_purchases
```
Comment (fivetran-catfritz) [inline models/intermediate/int_recharge__customer_details.sql:22]: These were shifted to the charges cte to make clear the differentiation between an order or charge source.

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-catfritz Looks good to merge!

**Pull request merged**